ajax/libs/esri-leaflet-related/2.0.0/EsriLeafletRelated.min.js

```diff
@@ -0,0 +1 @@
+// Error: Unexpected token: keyword (var)
```

Comment:
Why don't you get minified files under `dist` folder? BTW, you need to check if files are minified successfully.